meta-lmp-base/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -89,5 +89,5 @@ do_install:append() {
 	ln -sf ../systemd-timesyncd-update.service ${D}${systemd_system_unitdir}/sysinit.target.wants/systemd-timesyncd-update.service
 
 	# Remove systemd-boot as it is provided by a separated recipe and we can't disable via pkgconfig
-	rm -rf ${D}${nonarch_base_libdir}/systemd/boot
+	rm -rf ${D}${nonarch_libdir}/systemd/boot
```

Comment:
I think is better to fail when the directory doesn't exist and if something change in future this will rise an error

```
if ${@bb.utils.contains('EFI_PROVIDER','systemd-boot','true','false',d)}; then
    rm -r ${D}${nonarch_libdir}/systemd/boot
fi
```

Reply:
Yup, will update.

Reply:
Done.